task26.py

```diff
@@ -0,0 +1,8 @@
+def power(base, exp):
+    if (exp == 1):
+        return (base)
+    if (exp != 1):
+        return (base * power(base, exp - 1))
```

Comment:
И снова вездесущее скобки.
```suggestion
    if exp == 1:
        return base
    if exp != 1:
        return base * power(base, exp - 1)
```